how-to-participate-in-meetings.md

```diff
@@ -26,6 +26,28 @@ issues you see with it, and articulate them. What are the limitations? Are there
 champions can do to help their case? What sort of improvements are needed? Have requirements been
 missed? Try to move the discussion in a positive direction!
 
+### Procedure for establishing consensus
+
+Committee consensus is required to advance a proposal to a subsequent stage and to merge normative PRs. To achieve consensus in committee, take the following steps:
+1. Place the item, with a link to the detailed materials, on the TC39 agenda ahead of the agenda deadline.  This is to ensure everyone has a chance to review it ahead of the meeting, and insert schedule constraints for when they want to participate in the discussion. Note that some schedule constraints may force the topic into a future meeting.
+1. Discuss it at the meeting in a presentation, hearing out all concerns from the committee. If there is too much to discuss to fit within the timebox, then rather than jumping to asking for consensus, prefer to continue discussion in an overflow item or future meeting.
+1. When discussion has concluded, a call for consensus can be made, as follows:
+    1. The presenter asks the committee for consensus on $xyz.
+         1. (Note, the chair may encourage the presenter to make this request if it appears that the committee conversation has settled, or the presenter may decide to ask this directly.)
+    1. One TC39 attendee states, “I nominate $xyz for consensus.”
+    1. Another states, “I second $xyz for consensus.”
```

Comment:
requiring at least one explicit "yes" seems reasonable, but two, and the formalism, seems a bit overblown to me

Reply:
About formality: people in committee seem to have fun with formal language sometimes, so this proposal goes a little camp/extra with it all. I would be fine to tone that down.

The main element of this proposal is to say, two people in committee besides the champion should say, "I want this proposal to go forward" before we call for objections. I think that is quite a low bar, and would give more people a chance to constructively participate as well as say the reason they support it. This is just an idea to improve engagement in committee.

What if we said, one is technically enough if it came down to it, but two is encouraged?

Reply:
That seems fine; i certainly agree that ideally, there’s a multitude of explicit supporters.

Reply:
OK, I've edited the text to require just a minimum of one, and reduce the formality. Thanks for the approval! We'll discuss this in TC39 before it lands.

Reply:
Happy to discuss this in in TC39, but I think the original text requiring two people to explicitly support a proposal is a good idea and presents a very low bar, when we often have 50 or 60 people present in committee. I'm happy to see we're trying to increase engagement and move away from proposals getting consensus through lack of objections rather than explicit support.